app/messages/summaries.email.html.erb

```diff
@@ -11,17 +11,19 @@
 <% end %>
 
 <% define_content :footer_link do %>
+<!--
```

Comment:
> This thing yields back to the layout and fills the generic one
> on the left. It differs in various emails.
> 
> I'm keeping the communication profile url which is on the right of
> all emails.

Maybe I'm taking crazy pills, but it looks like you commented out the specific communication profile one and left the generic footer_link.  Or are you saying that the footer_link is the one that resolves to the communication profile?